ingest/sqs/pools/transformer/pool_transformer.go

```diff
@@ -427,3 +473,58 @@ func (pi *poolTransformer) computeUSDCPoolLiquidityCapFromUOSMO(ctx sdk.Context,
 
 	return poolLiquidityCapUOSMO, noPoolLiquidityCapError
 }
+
+func (pi *poolTransformer) updateAlloyTrasmuterInfo(
+	ctx sdk.Context,
+	pool poolmanagertypes.PoolI,
+	cwPool cosmwasmpooltypes.CosmWasmExtension,
+	cosmWasmPoolModel *sqsdomain.CosmWasmPoolModel,
+	denoms *[]string,
+) error {
+	bz, err := pi.wasmKeeper.QuerySmart(ctx, cwPool.GetAddress(), []byte(`{"list_asset_configs":{}}`))
+	if err != nil {
+		return fmt.Errorf(
+			"error querying list_asset_configs for pool (%d) contrat_address (%s): %w",
+			pool.GetId(), pool.GetAddress(), err,
+		)
+	}
+	var assetConfigsResponse struct {
+		AssetConfigs []sqsdomain.TransmuterAssetConfig `json:"asset_configs"`
+	}
```

Comment:
nit: let's please define types and JSON schemas at the top of the file for readability and ease of finding

Reply:
I think leaving this here is better, since it's only being used within the function. That fact even more obvious after extracting these queries into small functions. and only return the content inside, not is wrapped structure.